### ISSUE TITLE
Time to set samplerate

### ISSUE THREAD

**Issue by Dominik-Vogel** [QCoDeS/broadbean#70]
Currently the sample rate has to be specified for every Blueprint. It is desirable to build entire sequences without having to specify the sample rate and set it once it needs to be forged. 
We should think about the implications of setting the sample rate at that later stage.

**Comment by WilliamHPNielsen** [comment]
Yes, we should. It it stupid that we specify the same information several times.

When do we need a samplerate?

 - When we plot.
 - When we forge.
 - Possibly when we consistency check, but that should not really be necessary.

So I think it should be possible to either assign a sample rate to an object or simply pass it to the plotter/forger as a kwarg when needed. That actually goes for any object. It should then not be possible to have two objects with different sample rates inside the same (higher-ranking) object; that should be a rejection-at-insertion.
